CONTRIBUTING.md

```diff
@@ -57,7 +59,27 @@ early and often.
 
 We are using [conventional
 commits](https://www.conventionalcommits.org/en/v1.0.0/) so please format your
-commit messages accordingly.
+commit messages accordingly. There is a github action that uses the
+[gitlint](https://jorisroovers.com/gitlint/latest/) linter to check your commit
+messages for validity.
+
+Please try to use a scope when writing a commit message. APIS consists of
+various individual modules, so the scope tells the users which module your
+change refers to.
```

Comment:
Would add 1-2 examples, or link to a few actual commits to demonstrate how we use them.

One simple one and a combined one (multiples, separated by commas) would make sense.